charts/dm-store/values.yaml

```diff
@@ -68,6 +68,7 @@ java:
     #Below File size limits are in MB
     MAX_MEDIA_FILE_SIZE: 500
     MAX_NON_MEDIA_FILE_SIZE: 1024
+    MAX_DISALLOWED_FILE_SIZE: 0
```

Comment:
can it be different number?
basically we do not accept empty files and
is there any team which may need to upload empty files as a place holder?